api/awsauth/serverless.yml

```diff
@@ -41,6 +41,7 @@ plugins:
 
 provider:
   name: aws
+  region: us-east-1
```

Comment:
This fixes the original `${self:provider.region}` error, but after that was fixed I ran into a permissions error that I didn't want to untangle. (https://github.com/covid-projections/covid-data-model/runs/8105515757?check_suite_focus=true) 

So I just pinned the version.